### PR TITLE
createGroundPolylineGeometry dependencies, Scene cleanup

### PR DESCRIPTION
**Pull request by kring**
Hopefully fairly uncontroversial stuff here from our Cesium fork:

* Remove dependency of `createGroundPolylineGeometry` on `GroundPolylinePrimitive`. It was only needed for initializing terrain heights, which was delegated to `ApproximateTerrainHeights` anyway. This change should make that web worker smaller and in my webpack configuration, at least, it avoids a dependency cycle that makes the build hang.
* Make `Scene` more consistent in how it destroys sub-objects. Random, inconsequential, but overall positive change I had kicking around.

I don't these things need to be mentioned in CHANGES.md, but let me know if you disagree.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @kring!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/minor-tweaks/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: Source/Scene/GroundPolylinePrimitive.js [hunk 1]
```diff
@@ -343,27 +343,14 @@ define([
...
-            return initPromise;
-        }
-
-        GroundPolylinePrimitive._initPromise = ApproximateTerrainHeights.initialize()
-            .then(function() {
-                GroundPolylinePrimitive._initialized = true;
-            });
-
-        return GroundPolylinePrimitive._initPromise;
+        return ApproximateTerrainHeights.initialize();
```
Comment (hpinkos) [inline Source/Scene/GroundPolylinePrimitive.js]: Should we make this same change to `GroundPrimitive.initializeTerrainHeights`?

**Comment by hpinkos** [comment]
That's my only comment, thanks @kring!

**Comment by kring** [comment]
> @kring is this generally helpful for anyone using webpack? If so, is it worth a mention in CHANGES.md?

I'm not sure. The fun thing about webpack is it has a billion configuration options, so other webpack configurations might not have the same problem. I'll add a note though, can't hurt.

**Comment by kring** [comment]
This is ready for another look.

**Comment by hpinkos** [comment]
Thanks @kring!

**Pull request merged**